src/PIL/ImageFile.py

```diff
@@ -31,6 +31,7 @@
 import itertools
 import struct
 import sys
+from typing import List
```

Comment:
Let's add `from __future__ import annotations` as the first import and remove this:
```suggestion
```